packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -98,6 +139,117 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
     });
   }
 
+  private fetchAptosWallets() {
+    let { aptosWallets, on } = getAptosWallets();
+    this.setWallets(aptosWallets);
+
+    if (typeof window === "undefined") return;
```

Comment:
Just thoughts for later, we should probably put these into a reusable function, since we check that the window is undefined lots of places.